addons/web/static/src/views/fields/many2many_binary/many2many_binary_field.xml

```diff
@@ -4,22 +4,22 @@
     <t t-name="web.Many2ManyBinaryField" owl="1">
         <div t-attf-class="oe_fileupload {{props.className ? props.className : ''}}" aria-atomic="true">
             <div class="o_attachments">
-                <t t-foreach="state.files" t-as="file" t-key="file_index">
-                   <t t-if="true" t-call="Many2ManyBinaryField.attachment_preview"/>
+                <t t-foreach="files" t-as="file" t-key="file_index">
+                   <t t-call="Many2ManyBinaryField.attachment_preview"/>
                 </t>
             </div>
             <div t-if="!props.readonly" class="oe_add">
-                <FileUploader
+                <FileInput
                     acceptedFileExtensions="props.acceptedFileExtensions"
                     multiUpload="true"
-                    onUploaded.bind="onFileUploaded"
+                    onUpload.bind="onFileUploaded"
+                    resModel="'ir.attachment'"
+                    resId="props.record.resId or props.record.context.uid"
```

Comment:
@aab-odoo Does it sounds right to you? It no longer crashes with this. I'm not sure about what the resId must be for the FileInput